docs/user/summary.md

```diff
@@ -0,0 +1,1115 @@
+  Mapping deviations from source content type to Nautobot content type
+sessions.session | Disabled with reason: Nautobot has own sessions, sessions should never cross apps.
+admin.logentry | Disabled with reason: Not directly used in Nautobot.
+users.userconfig: Nautobot Model Not Found
+auth.permission | Disabled with reason: Handled via a Nautobot model and may not be a 1 to 1.
+auth.user => users.user
+dcim.sitegroup => dcim.locationtype
+dcim.region => dcim.location
+dcim.site => dcim.location
+dcim.cablepath | Disabled with reason: Recreated in Nautobot on signal when circuit termination is created
+dcim.rackrole => extras.role
+dcim.cabletermination_a => dcim.cable
+dcim.cabletermination_b => dcim.cable
+dcim.devicerole => extras.role
+ipam.role => extras.role
+ipam.aggregate => ipam.prefix
+dcim.modulebay: Nautobot Model Not Found
+dcim.modulebaytemplate: Nautobot Model Not Found
+dcim.moduletype: Nautobot Model Not Found
+dcim.module: Nautobot Model Not Found
+ipam.asn: Nautobot Model Not Found
+ipam.iprange: Nautobot Model Not Found
+tenancy.contactgroup: Nautobot Model Not Found
+tenancy.contactrole: Nautobot Model Not Found
+tenancy.contact: Nautobot Model Not Found
+tenancy.contactassignment: Nautobot Model Not Found
+dcim.inventoryitemrole: Nautobot Model Not Found
+dcim.inventoryitemtemplate: Nautobot Model Not Found
+dcim.cabletermination: Nautobot Model Not Found
+dcim.virtualdevicecontext: Nautobot Model Not Found
+ipam.fhrpgroup: Nautobot Model Not Found
+ipam.fhrpgroupassignment: Nautobot Model Not Found
+ipam.servicetemplate: Nautobot Model Not Found
+ipam.l2vpn: Nautobot Model Not Found
+ipam.l2vpntermination: Nautobot Model Not Found
+extras.report: Nautobot Model Not Found
+extras.script: Nautobot Model Not Found
+extras.journalentry: Nautobot Model Not Found
+extras.configrevision: Nautobot Model Not Found
+extras.savedfilter: Nautobot Model Not Found
+extras.cachedvalue: Nautobot Model Not Found
+extras.branch: Nautobot Model Not Found
+extras.stagedchange: Nautobot Model Not Found
+users.adminuser: Nautobot Model Not Found
+wireless.wirelesslangroup: Nautobot Model Not Found
+wireless.wirelesslan: Nautobot Model Not Found
+wireless.wirelesslink: Nautobot Model Not Found
+django_rq.queue: Nautobot Model Not Found
+secrets.secret: Nautobot Model Not Found
+secrets.secretrole: Nautobot Model Not Found
+secrets.userkey: Nautobot Model Not Found
+secrets.sessionkey: Nautobot Model Not Found
+```
+
+## Content Types Back Mapping
+
+This section shows back mapping deviations from Nautobot content types to NetBox content types. Only those content types that differ between NetBox and Nautobot are shown.
+
+"Ambiguous" means that there are multiple NetBox content types that map to the same Nautobot content type.
+
+```
+- Content Types Back Mapping: --------------------------------------------------
+  Back mapping deviations from Nautobot content type to the source content type
+extras.role => Ambiguous
+users.user => auth.user
```

Comment:
why is this in this list if not "Ambiguous"?

Reply:
This list contain all deviations, mapping to the different content type is considered a deviation.